lnrpc/lightning.proto

```diff
@@ -1510,6 +1510,9 @@ message Channel {
 
     // This is the confirmed / on-chain zero-conf SCID.
     uint64 zero_conf_confirmed_scid = 33;
+
+    // The alias of the remote node
+    string alias = 34;
```

Comment:
"alias" could mean either channel alias or peer alias. Would be better to be explicit about which one this is.